_docs/tasks/rate-limiting.md

```diff
@@ -53,16 +62,18 @@ Using Istio we can ensure that `1qps` is not breached.
    ```
    `istioctl` sets configuration for `subject=ratings.default.svc.cluster.local`
 
-2. Generate load on the `productpage` with the following command:
+1. Generate load on the `productpage` with the following command:
 
    ```bash
    while true; do curl -s -o /dev/null http://$GATEWAY_URL/productpage; done
    ```
-   
-   If you refresh the `productpage` (http://$GATEWAY_URL/productpage) while the load 
-   generator is running (i.e., generating more than 5 req/s), the traffic generated by
-   your browser will be rate limited. When `reviews` service is unable to access `ratings`
-   service you stop seeing stars on the UI.
+
+1. Refresh the `productpage` in your browser.
+
+   While the load generator is running (i.e., generating more than 1 req/s), the traffic generated by
+   your browser will be rate limited. 
+   Notice that if you log in as user "jason" or any other user, the `reviews` service is unable to
+   access the `ratings` service, so you stop seeing stars, red or black.
 
```

Comment:
General feedback on the story: while this may have been the current flow, how about we generate load with user Jason (use curl -b user=jason...). This way we just have black stars or no stars.